https://issues.redhat.com/browse/ACM-18510--Add What's new entry for RHEM (#7617)
open-cluster-management/rhacm-docs: release_notes/acm_whats_new.adoc

== Clusters
//this is where edge will go because it is an ACM function used for cluster management

> * You can now manage your edge devices with the {rhem} that offers you a scalable, simple, and secure device management. Learn more about the {rhem} at link:../edge_manager/edge_mgr_intro.adoc#edge-mgr-intro[{rhem} (Technology Preview)].

Reviewer: ```suggestion
* **Technology Preview:** You can now manage your edge devices with the {rhem}, which offers you scalable, simple, and secure device management. Learn more about the {rhem} at link:../edge_manager/edge_mgr_intro.adoc#edge-mgr-intro[{rhem} (Technology Preview)].
```

Reviewer: This looks great. If they want to say more "wows" about it because it is the first what's new, you can add more examples or highlights, but it is nice this way, too. Will approve.
